array.c

```diff
@@ -4303,14 +4303,14 @@ ary_resize_smaller(VALUE ary, long len)
 /*
  *  call-seq:
  *    array.delete(obj) -> deleted_object
- *    array.delete {|obj| ... } -> deleted_object or block_return
+ *    array.delete {|nosuch| ... } -> deleted_object or block_return
```

Comment:
This line still needs `(obj)` after `delete`:

```suggestion
 *    array.delete(obj) {|nosuch| ... } -> deleted_object or block_return
```

Reply:
After ↑ it looks good to merge